README.md

```diff
@@ -55,26 +55,29 @@ The easiest way to compile the code and run the system locally is using [Docker]
 * [Install Docker](https://docs.docker.com/get-docker/)
 * [Install docker-compose](https://docs.docker.com/compose/install/)
 
-Don't forget to run the docker daemon!
+### Recommended post-install steps
+- [Enable the Docker daemon](https://docs.docker.com/engine/install/linux-postinstall/#configure-docker-to-start-on-boot)
+  - It is recommended to read the other instructions here to customize other features you may find useful
+- [Enable the ability to run docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
+- Alternatively, advanced users may wish to run Docker in [rootless mode](https://docs.docker.com/engine/security/rootless/)
 
 ## Build the container
 
 ```terminal
-$ cd opencbdc-tx                 # change to the project directory
-$ sudo -s                        # open a root shell (needed for docker)
-# docker build . -t opencbdc-tx  # build the container
+# docker-compose -f docker-compose-2pc.yml build
```

Comment:
Here, and below, these commands are intended to be run as non-root, so they should be replaced with a `$` (Note though that the `#` inside the container is still correct and should be left as-is).